client/components/shared/styles.js

```diff
@@ -13,8 +14,12 @@ export const HugeText = styled.h2`
 export const Heading1 = styled.h1`
   font-style: normal;
   font-weight: bold;
-  font-size: 53px;
-  line-height: 120%;
+  font-size: 35px;
```

Comment:
Remember that this changes the styling wherever we use this text (also for future references). We just need to be sure that we want to change the general styling of the h1, and not on a particular page 🙂

Reply:
@Soreg I fought with this for a while but when on a mobile device the H1 was just too large. The Figma  designs have it smaller as well (reduced to an H3 there for reference). 
Let me know if you feel it should be larger / handled differently.